string property
string value
bool data
---
string[] instances
